frontend/src/components/unordered-list/presets.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { UnorderedList } from '.'
+import { css } from '@emotion/core'
+
+export const Ul = props => (
+  <UnorderedList
+    fontSize={[2, null, 3]}
+    lineHeight={[2, null, 3]}
+    pl={[5, null, 6]}
+    mb={4}
+    {...props}
+  >
+    {props.children}
+  </UnorderedList>
+)
+
+Ul.propTypes = {
+  children: PropTypes.any,
+}
+
+export const UlNone = props => (
+  <UnorderedList
+    fontSize={[2, null, 3]}
+    lineHeight={[2, null, 3]}
+    mb={4}
+    css={css`
+      list-style-type: none;
```

Comment:
why not just have `Ul` and have the `list-style-type` customizable?

Reply:
Could definitely be done, but would require some more logic in the UnorderedList component since styled-components doesn't gift us with a prop to handle it (sad). I have a vague recollection of trying, hitting a road block, and then doing this instead. Definitely open to refactoring after this PR.

Oh, also, because UlNone would require it's own preset without margins anyway. Hmm 🤔